src/Intl/Icu/README.md

```diff
@@ -0,0 +1,19 @@
+Symfony Polyfill / Intl: ICU
+============================
+
+This component maps the following functions to the
+[`symfony/intl`](https://github.com/symfony/intl) component in the abscense of the
+[Intl](http://php.net/intl) extension.
+
+- `intl_is_failure($errorCode)` -> `IntlGlobals::isFailure($errorCode)`
+- `intl_get_error_code()` -> `IntlGlobals::getErrorCode()`
+- `intl_get_error_message()` -> `IntlGlobals::getErrorMessage()`
+- `intl_error_name($errorCode)` -> `IntlGlobals::getErrorName($errorCode)`
```

Comment:
also the `Collator`, `NumberFormatter`, `Locale` and `IntlDateFormatter` classes (thx to symfony/intl for now)
I wouldn't tell about the `IntlGlobals` class, this is an internal implementation detail